### ISSUE TITLE
style-loader support

### ISSUE THREAD

**Issue by MitchTalmadge** [webpack-contrib/css-minimizer-webpack-plugin#80]
Documentation Is:

- [ ] Missing
- [ ] Needed
- [x] Confusing
- [ ] Not Sure?

### Please Explain in Detail...

I am unsure if this plugin supports `style-loader`. In my attempts to get it working, I have not seen any differences between using this plugin and not using it; however, I am unsure if this is my fault or because the plugin does not support it.

### Your Proposal for Changes

Document whether this plugin works for `style-loader` or if it requires something like `mini-css-extract-plugin`.

**Comment by alexander-akait** [comment]
No, right now you need use `cssnano` as `postcss` plugin, but let's merge this as feature for better DX

**Comment by tunnelpuzzle** [comment]
@alexander-akait What would be the config to make `cssnano` and `postcss-loader` work with `style-loader`?

**Comment by alexander-akait** [comment]
@tunnelpuzzle Currently you can use `cssnano` as a plugin for postcss for style which use `style-loader`

**Comment by tunnelpuzzle** [comment]
@alexander-akait Is it possible to do that if you're also using `css-loader`? Not using `css-loader` that works but adding `css-loader` seems to make it not minify.
